cmd/finch/nerdctl.go

```diff
@@ -173,7 +173,7 @@ func (nc *nerdctlCommand) run(cmdName string, args []string) error {
 
 	var additionalEnv []string
 	switch cmdName {
-	case "build", "pull", "push":
+	case "build", "pull", "push", "run":
```

Comment:
We should also handle `image build`, `image pull`, `image push` and `container run` as they are the new command syntax for the above case

Reply:
good point, added the subcommands in latest revision. I won't create a system to more eloquently match these subcommands as I know some of that work has been done in the windev branch already